apps/institution-service/openApi.json

```diff
@@ -496,15 +496,15 @@
           },
           {
             "in": "query",
-            "name": "isActive",
+            "name": "includeInactiveIntegrations",
             "schema": {
               "type": "boolean",
               "enum": [true],
               "example": true
             },
             "required": false,
             "example": true,
-            "description": "Only show Institutions that are active"
+            "description": "Include institutions without active integrations in the search"
```

Comment:
When using aggregator integration filters include institutions that have inactive integrations that match the filters